tests/unit/utils/test_pkg.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+# Import Python libs
+from __future__ import absolute_import
+# Import Salt Libs
+import salt.utils.pkg
+# Import Salt Testing Libs
+from tests.support.unit import TestCase
+
+
+class PkgUtilsTestCase(TestCase):
+    '''
+    TestCase for salt.utils.pkg module
+    '''
+    test_parameters = [
+        ("16.0.0.49153-0+f1", "", "16.0.0.49153-0+f1"),
+        ("> 15.0.0", ">", "15.0.0"),
+        ("< 15.0.0", "<", "15.0.0"),
+        ("<< 15.0.0", "<<", "15.0.0"),
+        (">> 15.0.0", ">>", "15.0.0"),
+        (">= 15.0.0", ">=", "15.0.0"),
+        ("<= 15.0.0", "<=", "15.0.0"),
+        ("!= 15.0.0", "!=", "15.0.0"),
+        ("<=> 15.0.0", "<=>", "15.0.0"),
+        ("<> 15.0.0", "<>", "15.0.0"),
+        ("= 15.0.0", "=", "15.0.0"),
+        (">15.0.0", ">", "15.0.0"),
+        ("<15.0.0", "<", "15.0.0"),
+        ("<<15.0.0", "<<", "15.0.0"),
+        (">>15.0.0", ">>", "15.0.0"),
+        (">=15.0.0", ">=", "15.0.0"),
+        ("<=15.0.0", "<=", "15.0.0"),
+        ("!=15.0.0", "!=", "15.0.0"),
+        ("<=>15.0.0", "<=>", "15.0.0"),
+        ("<>15.0.0", "<>", "15.0.0"),
+        ("=15.0.0", "=", "15.0.0"),
+        ("", "", "")
+    ]
+
+    def test_split_comparison(self):
+        '''
+        Test passing a jid on the command line
+        '''
+        for test_parameter in PkgUtilsTestCase.test_parameters:
```

Comment:
There is no need to refer to class attributes here are they are inherited by the instance. You can use `self.test_parameters` here.